common/src/main/java/com/gentics/mesh/dagger/OverrideSupplier.java

```diff
@@ -0,0 +1,21 @@
+package com.gentics.mesh.dagger;
+
+import com.gentics.mesh.Mesh;
+
+/**
+ * A supplier function for the support of overridden Dagger dependencies.
+ * 
+ * @author plyhun
+ *
+ */
+@FunctionalInterface
```

Comment:
A main overridden dep supplier interface. Made to not interfere with the original deps.